libqtile/widget/rhythmbox.py

```diff
@@ -0,0 +1,117 @@
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+# warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with this program. If not, see
+# <http://www.gnu.org/licenses/>.
+
+import subprocess
+
+from functools import partial
+
+from libqtile import pangocffi
+from libqtile.widget import base
+
+
+class Rhythmbox(base.ThreadPoolText):
+    """
+    A simple Rhythmbox widget that is based on the Cmus widget.
+
+    Show metadata about the currently paused or playing song, and allow basic mouse control from the bar:
+        - Toggle pause (if playing) or play (if paused) on left click
+        - Skip forward in playlist on scroll up
+        - Skip backward in playlist on scroll down
+
+    The following options (extracted from ``man rhythmbox-client``) are available for the `format` string:
+
+    - ``%at``: album title
+    - ``%aa``: album artist
+    - ``%aA``: album artist (lowercase)
+    - ``%as``: album artist sortname
+    - ``%aS``: album artist sortname (lowercase)
+    - ``%ay``: album year
+    - ``%ag``: album genre
+    - ``%aG``: album genre (lowercase)
+    - ``%an``: album disc number
+    - ``%aN``: album disc number, zero padded
+    - ``%st``: stream title
+    - ``%tn``: track number (i.e., 8)
+    - ``%tN``: track number, zero padded (i.e., 08)
+    - ``%tt``: track title
+    - ``%ta``: track artist
+    - ``%tA``: track artist (lowercase)
+    - ``%ts``: track artist sortname
+    - ``%tS``: track artist sortname (lowercase)
+    - ``%td``: track duration
+    - ``%te``: track elapsed time
+
+    Rhythmbox (https://www.rhythmbox.org/) and Playerctl (https://github.com/altdesktop/playerctl) should be installed.
+
+    This widget supports the following bar orientations: horizontal and vertical.
+    """
+
+    defaults = [
+        ("format", "%aa - %tt", "Track information to display"),
+        ("play_icon", "", "Icon to display when track is playing"),
+        ("pause_icon", "", "Icon to display when track is paused"),
+        ("stop_icon", "", "Icon to display when track is stopped"),
+        ("play_color", "00ff00", "Text color when track is playing"),
+        ("no_play_color", "cecece", "Text color when track is paused or stopped"),
+        ("update_interval", 0.5, "Update time in seconds"),
+    ]
+
+    def __init__(self, **config):
+        base.ThreadPoolText.__init__(self, "", **config)
+        self.add_defaults(Rhythmbox.defaults)
+        self.status = ""
+
+        self.add_callbacks(
+            {
+                "Button1": self.play,
+                "Button4": partial(subprocess.Popen, ["rhythmbox-client", "--next"]),
+                "Button5": partial(subprocess.Popen, ["rhythmbox-client", "--previous"]),
```

Comment:
Using `Popen` and not `wait()`ing for the process to exit will create zombie processes.

```suggestion
                "Button4": lazy.spawn("rhythmbox-client --next"),
                "Button5": lazy.spawn("rhythmbox-client --previous"),
```